.github/workflows/agent-pr.yml

```diff
@@ -26,15 +26,14 @@ jobs:
           git checkout -b agent-${{ github.event.inputs.agent_version }}
           git push -u origin agent-${{ github.event.inputs.agent_version }}
       - name: Update Files
-        run: .github/scripts/update-agent-version.py ${{ github.event.inputs.agent_version }}
+        run: .github/scripts/update_versions.py set_app_version --version ${{ github.event.inputs.agent_version }}
       - name: Commit Changes
         uses: swinton/commit@v2.x
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           files: |
             charts/netdata/Chart.yaml
-            charts/netdata/README.md
```

Comment:
`README.md` will be update as part of the release process only to avoid pre mature update of the version before its actually been released